### ISSUE TITLE
亲爱的作者，你好，我依旧无法获取满意的结果。

### ISSUE THREAD

**Issue by kearcer** [JiaRenChang/PSMNet#191]
我采用了您提供的pretrained_model_kitti2015预训练模型进行微调,得到了很好的结果，但当我直接用你pretrained_model_kitti2015进行测试，我发现它在KITTI2015上的误差甚至只有2.28%。这证明了这本来就是一个已经微调过后的最终模型了。而我现在还想知道的是：
1：pretrained_scene_flow模型到pretrained_model_kitti2015模型的学习率和参数是多少，是您在github中给的200epoch0.01lr+100epoch+0.001lr？还是你在论文中提到的长周期1000个epoch得到的结果。
2：如果是1000个epoch得到的结果，那么我们lr应该怎么设置？你在github提供的300epoch应该获得的结果又是多少？
多次打扰到您非常抱歉，但是这确实对我来说非常重要，期待您的回答。万分感谢！

**Comment by JiaRenChang** [comment]
@kearcer 
The learning rate scheduling for 1000 epoch is that  0.001 is for the first 600 epoch as well as 0.0001 for the rest of epoch.

**Comment by kearcer** [comment]
感谢，我已经解决了，因为设置了验证集导致部分场景无法充分训练导致了测试集输出结果部分场景效果很差
